Generator/DoctrineFixtureGenerator.php

```diff
@@ -153,12 +153,12 @@ public function getFixtureNameFromEntityName($entity, array $ids = [], $prefix =
     {
         // Throw error if $entity is not a string
         if (!is_string($entity)) {
-            throw new \InvalidArgumentException('The parameter $entity is expected to be a string, "' . typeof($entity) . '" given.');
+            throw new \InvalidArgumentException('The parameter $entity is expected to be a string, "' . gettype($entity) . '" given.');
```

Comment:
typeof was a typo, as I'm used to use it in C programming, but it's not a php function.